### PR TITLE
debug: make the marshalled JSON from backup meta more human readable.

### PR DESCRIPTION
**Pull request by YuJuncen**
### What problem does this PR solve? 
Make the result of `br debug backupmeta decode` more readable.

### What is changed and how it works?
Mapped some `[byte]` fields to `string`, with hex encoding or simply encode with UTF-8.

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

diff:
```diff
diff --git a/backupmeta.old.json b/backupmeta.new.json
index a61aa33..81ca02a 100644
--- a/backupmeta.old.json
+++ b/backupmeta.new.json
@@ -1,62 +1,62 @@
 {
-  "cluster_id": 6946469498797568000,
-  "cluster_version": "\"5.0.0-rc.x\"\n",
   "files": [
     {
+      "sha256": "5759c4c73789d6ecbd771b374d42e72a309245d31911efc8553423303c95f22c",
+      "end_key": "7480000000000000ff0500000000000000f8",
       "name": "1_4_2_default.sst",
-      "sha256": "V1nExzeJ1uy9dxs3TULnKjCSRdMZEe/IVTQjMDyV8iw=",
-      "end_key": "dIAAAAAAAAD/BQAAAAAAAAD4",
       "total_kvs": 153,
       "total_bytes": 824218,
       "cf": "default",
       "size": 44931
     },
     {
+      "sha256": "87597535ce0edbc9a9ef124777ad1d23388467e60c0409309ad33af505c1ea5b",
+      "start_key": "7480000000000000ff0f00000000000000f8",
+      "end_key": "7480000000000000ff1100000000000000f8",
       "name": "1_16_8_58be9b5dfa92efb6a7de2127c196e03c5ddc3dd8ff3a9b3e7cd4c4aa7c969747_1617689203876_default.sst",
-      "sha256": "h1l1Nc4O28mp7xJHd60dIziEZ+YMBAkwmtM69QXB6ls=",
-      "start_key": "dIAAAAAAAAD/DwAAAAAAAAD4",
-      "end_key": "dIAAAAAAAAD/EQAAAAAAAAD4",
       "total_kvs": 1,
       "total_bytes": 396,
       "cf": "default",
       "size": 1350
     },
     {
+      "sha256": "97bd1b07f9cc218df089c70d454e23c694113fae63a226ae0433165a9c3d75d9",
+      "start_key": "7480000000000000ff1700000000000000f8",
+      "end_key": "7480000000000000ff1900000000000000f8",
       "name": "1_24_12_72fa67937dd58d654197abadeb9e633d92ebccc5fd993a8e54819a1bd7f81a8c_1617689203853_default.sst",
-      "sha256": "l70bB/nMIY3wiccNRU4jxpQRP65joiauBDMWWpw9ddk=",
-      "start_key": "dIAAAAAAAAD/FwAAAAAAAAD4",
-      "end_key": "dIAAAAAAAAD/GQAAAAAAAAD4",
       "total_kvs": 35,
       "total_bytes": 761167,
       "cf": "default",
       "size": 244471
     },
     {
+      "sha256": "6dcb6ba2ff11f4e7db349effc98210ba372bebbf2470e6cd600ed5f2294330e7",
+      "start_key": "7480000000000000ff3100000000000000f8",
+      "end_key": "7480000000000000ff3300000000000000f8",
       "name": "1_50_25_2f1abd76c185ec355039f5b4a64f04637af91f80e6cb05099601ec6b9b1910e8_1617689203867_default.sst",
-      "sha256": "bctrov8R9OfbNJ7/yYIQujcr678kcObNYA7V8ilDMOc=",
-      "start_key": "dIAAAAAAAAD/MQAAAAAAAAD4",
-      "end_key": "dIAAAAAAAAD/MwAAAAAAAAD4",
       "total_kvs": 22,
       "total_bytes": 1438283,
       "cf": "default",
       "size": 1284851
     },
     {
+      "sha256": "ba603af7ecb2e21c8f145d995ae85eea3625480cd8186d4cffb53ab1974d8679",
+      "start_key": "7480000000000000ff385f72ffffffffffffffffff0000000000fb",
       "name": "1_2_33_07b745c3d5a614ed6cc1cf21723b161fcb3e8e7d537546839afd82a4f392988c_1617689203895_default.sst",
-      "sha256": "umA69+yy4hyPFF2ZWuhe6jYlSAzYGG1M/7U6sZdNhnk=",
-      "start_key": "dIAAAAAAAAD/OF9y////////////AAAAAAD7",
       "total_kvs": 260000,
       "total_bytes": 114425025,
       "cf": "default",
       "size": 66048845
     }
   ],
-  "is_raw_kv": true,
   "raw_ranges": [
     {
       "cf": "default"
     }
   ],
-  "ddls": "bnVsbA==",
+  "ddls": "null",
+  "cluster_id": 6946469498797568000,
+  "cluster_version": "\"5.0.0-rc.x\"\n",
+  "is_raw_kv": true,
   "br_version": "BR\nRelease Version: v5.0.0-master\nGit Commit Hash: c0d60dae4998cf9ac40f02e5444731c15f0b2522\nGit Branch: HEAD\nGo Version: go1.13.4\nUTC Build Time: 2021-03-25 08:10:08\nRace Enabled: false"
 }
```

### Release Note

 - No release note. 

(Since this command is only for internal usage.)

### CODE REVIEW

File: pkg/utils/json_test.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "db": "{\"id\":1,\"db_name\":{\"O\":\"test\",\"L\":\"test\"},\"charset\":\"utf8mb4\",\"collate\":\"utf8mb4_bin\",\"state\":5}",
```
Comment (kennytm) [inline pkg/utils/json_test.go:34]: is it possible to further decode this as a JSON object?

**Review by kennytm — COMMENTED** [review]
rest LGTM

**Comment by YuJuncen** [comment]
`[2021-04-06T06:56:36.895Z] Error: restore table `cppk_tsr`.`tbl2` failed: rpc error: code = Unknown desc = FileExists("/tmp/backup_restore_test/importer/57dbfa8a-c8e0-5b52-90f9-8db28a8a4ff1")
`

New CI failure? 🤔

/run-integration-tests

File: pkg/utils/json_test.go (hunk 1 above)
Comment (YuJuncen) [reply pkg/utils/json_test.go:34]: Maybe `map[string]interface{}` could help. Let me try. 🤔

Comment (YuJuncen) [reply pkg/utils/json_test.go]: Done, pal

**Pull request merged**